simple_centralized_greedy.py

```diff
@@ -0,0 +1,69 @@
+# This file implement simplified greedy replication algorithm.
+
+# Python import
+import random
+import os
+import sys
+import time
+
+# Project imports
+sys.path.insert(0, os.path.join(os.path.dirname(os.path.realpath(__file__)), 'aggregator'))
+import util
+from aggregator import Aggregator
+
+class SimpleCentralizedGreedy:
+
+  def __init__(self):
+    self.aggregator = Aggregator() # to retrive server logs
+    self.server_set = set(util.retrieve_server_list()) # [server_ip, ]
+    self.content_set = set([])
+    self.replica_map = {} # { file_uuid: servers that store the file }
+    self.replication_task = []
+    self.last_timestamp = 0 # the time stamp of last update
+  
+  def update(self):
+    # update inner data
+    self.replica_map = {}
+    self.replication_task = []
+    # update content_set, replica_map
+    for server in self.server_set:
+      file_list = util.get_file_list_on_server(server)
+      for file_uuid in file_list:
+        self.content_set.add(file_uuid)
+        if file_uuid not in self.replica_map:
+          self.replica_map[file_uuid] = []
+        if server not in self.replica_map[file_uuid]:
+          self.replica_map[file_uuid].append(server)
+
+    current_timestamp = int(time.time())
```

Comment:
yes. we assume all timestamps are utc time. though time.time() is platform specific, it usually returns UTC time.